signed/sign.go

```diff
@@ -7,11 +7,19 @@ import (
 )
 
 func Sign(s *data.Signed, k *data.Key) {
+	id := k.ID()
+	signatures := make([]data.Signature, 0, len(s.Signatures))
```

Comment:
No, because I don't append the new signature to this slice, this is just the existing signatures